.github/workflows/main.yml

```diff
@@ -31,8 +31,7 @@ jobs:
             rust: stable
           - build: msrv
             os: ubuntu-latest
-            # sync MSRV with docs: guide/src/guide/installation.md
-            rust: 1.54.0
+            rust: 1.56.0
```

Comment:
It looks like there was a comment removed from here. Can we make sure it is retained?

Reply:
Definitely, it should be fixed now.